Server/src/models/interfaces/DataRequestModelInterface.ts

```diff
@@ -0,0 +1,9 @@
+export interface IDataRequestModel {

```

Comment:
I like this model for data requests

Reply:
Will talk with frontend to figure out how many materials they want maximum in a search, this can be expanded.

Reply:
As of commit 529a1d5 the GET request now accepts an array of materials, allowing for multi-material search functionality.